### PR TITLE
add superscript and subscript styles

### PR DESCRIPTION
**Pull request by huqigu**
I have added the superscript and subscript styles, please check for any bugs

### CODE REVIEW

File: lib/src/translations/toolbar.i18n.dart [hunk 1]
```diff
@@ -40,6 +40,8 @@ extension Localization on String {
           'Font family': 'Font family',
           'Font size': 'Font size',
           'Bold': 'Bold',
+          'Subscript': 'Subscript',
```
Comment (singerdmx) [inline lib/src/translations/toolbar.i18n.dart:43]: Add these for all locales

Comment (huqigu) [reply lib/src/translations/toolbar.i18n.dart:43]: > Add these for all locales

Do you need me to translate all languages？

File: lib/src/models/documents/attribute.dart [hunk 2]
```diff
@@ -359,7 +367,7 @@ class TokenAttribute extends Attribute<String> {
 // `script` is supposed to be inline attribute but it is not supported yet
```
Comment (singerdmx) [inline lib/src/models/documents/attribute.dart:367]: Delete this line

File: lib/src/widgets/default_styles.dart [hunk 3]
```diff
@@ -1,4 +1,5 @@
 import 'package:flutter/material.dart';
+import 'dart:ui';
```
Comment (singerdmx) [inline lib/src/widgets/default_styles.dart:2]: Why is it needed?

Comment (huqigu) [reply lib/src/widgets/default_styles.dart:2]: Done.

Comment (huqigu) [reply lib/src/widgets/default_styles.dart:2]: > Why is it needed?

to use FontFeature.subscripts()  and FontFeature.superscripts()

File: lib/src/models/documents/attribute.dart (hunk 2 above)
Comment (huqigu) [reply lib/src/models/documents/attribute.dart]: Done.

**Pull request merged**